### PR TITLE
DOC: clarify description of the "install" interface to reflect its convoluted behavior

### PR DESCRIPTION
**Pull request by yarikoptic**
Prompted by #7441 and might even close it if @asmacdo acdo agrees that it does.

I also changed "recommended" to "possible" in the paragraph about 'description' since I think we converged in some discussion in datalad-handbook that it is typically not recommended.

Now `--help` would look like

```shell
Usage: datalad install [-h] [-s URL-OR-PATH] [-d DATASET] [-g]
                       [-D DESCRIPTION] [-r] [-R LEVELS]
                       [--reckless [auto|ephemeral|shared-...]] [-J NJOBS]
                       [--branch BRANCH] [--version]
                       [URL-OR-PATH ...]

Install one or many datasets from remote URL(s) or local PATH source(s).

This command creates local sibling(s) of existing dataset(s) from
(remote) locations specified as URL(s) or path(s). Optional recursion into
potential subdatasets, and download of all referenced data is supported.
The new dataset(s) can be optionally registered in an existing
superdataset by identifying it via the DATASET argument (the new
dataset's path needs to be located within the superdataset for that).

If no explicit -s|--source option is specified, then all positional
URL-OR-PATH arguments are considered to be "sources" if they are URLs
or target locations if they are paths. If a target location path
corresponds to a submodule, the source location for it is figured out
from its record in the `.gitmodules`. If -s|--source is specified,
then a single optional positional PATH would be taken as the
destination path for that dataset.

It is possible to provide a brief description to label the dataset's
nature *and* location, e.g. "Michael's music on black laptop". This helps
...
```

### CODE REVIEW

File: datalad/distribution/install.py [hunk 1]
```diff
@@ -61,16 +61,27 @@
 
 @build_doc
 class Install(Interface):
-    """Install a dataset from a (remote) source.
+    """Install one or many datasets from remote URL(s) or local PATH source(s).
 
-    This command creates a local :term:`sibling` of an existing dataset from a
-    (remote) location identified via a URL or path. Optional recursion into
+    This command creates local :term:`sibling`(s) of existing dataset(s) from
```
Comment (adswa) [inline datalad/distribution/install.py:66]: there is a docbuild failure:

```

Warning, treated as error:
/opt/hostedtoolcache/Python/3.7.17/x64/lib/python3.7/site-packages/datalad/distribution/install.py:docstring of datalad.distribution.install.Install.__call__:1:term not in glossary: 'sibling`(s) of existing dataset(s) from (remote) locations specified as URL(s) or path(s). Optional recursion into potential subdatasets, and download of all referenced data is supported. The new dataset(s) can be optionally registered in an existing :term:`superdataset'
make: *** [Makefile:50: html] Error 2
make: Leaving directory '/home/runner/work/datalad/datalad/docs'
Error: Process completed with exit code 2.
```

I think the plural s needs to be escaped like this (untested, but often done in the handbook):
```suggestion
    This command creates local :term:`sibling`\(s) of existing dataset(s) from
```

Comment (yarikoptic) [reply datalad/distribution/install.py]: Thank you!

**Comment by yarikoptic** [comment]
we need a release with prior PR merged/resolving the issue.  So I marked this for a candidate to kick out the release

**Review by asmacdo — APPROVED** [review]
+1 this closes my issue.

**Comment by yarikoptic** [comment]
@mih or @adswa - any blessings (like clicking "Merge") or concerns?  if none - I would merge tomorrow.

**Comment by yarikoptic-gitmate** [comment]
PR released in [`0.19.2`](https://github.com/datalad/datalad/releases/tag/0.19.2)

**Pull request merged**